src/rpc/core_rpc_server.cpp

```diff
@@ -2593,9 +2594,27 @@ void core_rpc_server::invoke(BLS_EXIT_LIQUIDATION_LIST& rpc, rpc_context) {
                 assert(it != contrib_it.end());
                 contrib_it.erase(it);
             }
+
+            // NOTE: Remove operator cryptonote address and replace with ethereum address
+            std::string_view eth_address_key = "ethereum_address";
+            auto eth_address_it = contrib_it.find(eth_address_key);
+            if (eth_address_it != contrib_it.end()) {
+                auto eth_address = eth_address_it->template get<std::string>();
+                contrib_it.erase(eth_address_it);
+                contrib_it["address"] = eth_address;
+            }
         }
 
-        // NOTE: Remove implementation details from the contributor JSON
+        // NOTE: Remove operator cryptonote address and replace with ethereum address
+        std::string_view eth_address_key = "operator_ethereum_address";
+        auto eth_address_it = sn_info.find(eth_address_key);
+        if (eth_address_it != sn_info.end()) {
+            auto eth_address = eth_address_it->template get<std::string>();
+            sn_info.erase(eth_address_it);
+            sn_info["operator_address"] = eth_address;
+        }
```

Comment:
Some DRYness for these two chunks:
```C++
static void rename_key(nlohmann::json& obj, std::string_view old_key, std::string_view new_key) {
    if (auto it = obj.find(old_key); it != obj.end()) {
        auto val = std::move(it->second);
        obj.erase(it);
        obj[std::string{new_key}] = std::move(val);
    }
}
```
and then these two chunks become just:
```C++
rename_key(contrib_it, "ethereum_address", "address");
```
and
```C++
rename_key(sn_info, "operator_ethereum_address", "operator_address");
```

Reply:
Done